apple/Sources/FerrostarMapLibreUI/Views/NavigationMapView.swift

```diff
@@ -41,15 +47,20 @@ public struct NavigationMapView: View {
         camera: Binding<MapViewCamera>,
         navigationState: NavigationState?,
         onStyleLoaded: @escaping ((MLNStyle) -> Void),
-        @MapViewContentBuilder _ makeMapContent: () -> [StyleLayerDefinition] = { [] }
+        @MapViewContentBuilder makeMapContent: () -> [StyleLayerDefinition] = { [] },
+        mapViewModifiersWhenNotNavigating: @escaping (MapView<MLNMapViewController>) -> AnyView = { transferView in
+            AnyView(transferView)
```

Comment:
I wish we wouldn't have to type-erase here, but I didn't find a way to get this to work with it returning some View or a MapView<MLNMapViewController>  as this closure may contain both MapView modifiers such as onMapTapGesture, but also View modifiers such as onTapGesture.